modules/collection/programs/alacritty.nix

```diff
@@ -34,8 +34,8 @@ in {
       };
       description = ''
         The configuration converted into TOML and written to
-        `${config.directory}/.config/alacritty/alacritty.toml`.
-        Please reference https://alacritty.org/config-alacritty.html
+        `~/.config/alacritty/alacritty.toml`.
+        Please reference [https://alacritty.org/config-alacritty.html](https://alacritty.org/config-alacritty.html)
```

Comment:
You might be able to use automatic links here, e.g., `[](link)` but I don't undeerstand why the link body and the title are the same here?

Reply:
That does not render *yet* with `ndg`. I did this because I just want to render the link with the same text,  but I did not know you could omit the brackets part. It completely hides the link though, with ndg's parsing.

Reply:
That is a bug, please report all missing nixpkgs flavors.